source/localizable/cps_orders_api/_order_object.html.md.erb

```diff
@@ -0,0 +1,49 @@
+#### Order object
+
+Name                     | Type    | Description
+------------------------ | ------- | -----------------
+`code`                   | String  | Order code
+`date`                   | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order creation date
+`revenue`                | Double  | Total order price in euros
+`commission`             | Double  | Total order commission in euros
+`state`                  | String  | The current state of the order
+`rejection_reason`       | String  | The declined reason of the order, if it has been previously rejected
+`charged`                | Boolean | Whether the order has already been charged
+`charged_at`             | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order charge date (if has been charged)
+`refunded_at`            | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order refund date (if has been refunded)
+`line_items`             | Array   | [Order line items (products)](#order-line-items-array)
+`reject_options`         | Object  | [Reject options](#reject-options) (available for orders at a rejectable state)
+
+#### Order line items array
+
+Name                              | Type    | Description
+--------------------------------- | ------- | ------------------------------------------------
+`line_items[_].product_id`        | String  | Item id in shop
+`line_items[_].name`              | String  | Item product name
+`line_items[_].quantity`          | Integer | Item quantity
+`line_items[_].unit_price`        | Double  | Price per item in euros
```

Comment:
Perhaps we should say `Price per item unit in euros` since we refer to item as a total of units.

Reply:
Fixed.